### DISCUSSION TITLE
Hide the Copy and Paste buttons?

### DISCUSSION THREAD

**Discussion by Kaitydid789** [cortexrd/Knack-Toolkit-Library#277] [Q&A] [answered]
Hi there,

Is it possible to hide the Copy and Paste buttons when using the Bulk Edit function? I want people to just follow the instructions I put above the table. Please see the image below. 

Thank you!

<img width="1028" alt="Screen Shot 2024-04-05 at 5 59 54 AM" src="https://github.com/cortexrd/Knack-Toolkit-Library/assets/162930343/57ba3641-514d-4dd1-9a59-f638d1e7d8a8">

**Comment by cortexrd** [comment, marked as answer]
Hi @Kaitydid789,

First, you need to decide if you want to hide the Bulk Ops buttons in all pages, or just this particular view.

Per view:

`_cls=[ktlHidden], [ktlTarget, $('#view_1 .bulkOpsControlsDiv')]`

Replace `view_1` with yours.

All views, use this code in your Javascript pane:

```
$(document).on('KTL.BulkOperation.Updated', () => {
    $('.bulkOpsControlsDiv').addClass('ktlHidden');
});
```

Norm

**Reply by Kaitydid789** [reply]
It worked!! Went with just the view for now. Thanks so much for the quick response.

**Comment by cortexrd** [comment]
Always happy to help out.

Can you click the "Mark as answer" button please?


![image](https://github.com/cortexrd/Knack-Toolkit-Library/assets/42851359/7b704efb-6fed-451e-bcbc-c8581b6e2da5)


Norm
